pkg/db/migrations/202311151850_add_resources.go

```diff
@@ -13,6 +13,7 @@ func addResources() *gormigrate.Migration {
 		Source     string         `gorm:"index"`
 		ConsumerID string         `gorm:"index"`
 		Version    int            `gorm:"not null"`
+		Type       string         `gorm:"index"`
```

Comment:
can we add some comments here? It is not clear to me what data structure is stored in DB.

Reply:
added comments.